docs/ref/templates/builtins.txt

```diff
@@ -711,6 +711,12 @@ available to the included template::
     been evaluated and rendered* - not blocks that can be overridden by, for
     example, an extending template.
 
+.. note::
+    If the included template is missing, the effect varies depending on the
+    :setting:`DEBUG` setting: when ``DEBUG`` mode is on, a
+    ``TemplateDoesNotExist`` exception will be raised; otherwise a blank space
```

Comment:
"otherwise `{% include %}` silences the exception and returns an empty string."
Also would be helpful to note this behavior applies to *any* exception that's raised while rendering the included template.